ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/vxlan-interface.j2

```diff
@@ -1,21 +1,20 @@
 {# eos- VxLAN interface #}
-{% if vxlan_tunnel_interface is defined %}
+{% if vxlan_tunnel_interface is arista.avd.defined %}
 !
 interface Vxlan1
+{%     if vxlan_tunnel_interface.Vxlan1.source_interface is arista.avd.defined %}
    vxlan source-interface {{ vxlan_tunnel_interface.Vxlan1.source_interface }}
+{%     endif %}
 {%     if vxlan_tunnel_interface.Vxlan1.virtual_router is defined %}
```

Comment:
I believe we should update this to arista.avd.defined since we always expect a value.